src/src.pro

```diff
@@ -18,8 +18,8 @@
 #    59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             #
 ############################################################################
 
-lessThan(QT_MAJOR_VERSION, 5) {
-  error("requires Qt 5.0 or later")
+lessThan(QT_MAJOR_VERSION, 6):lessThan(QT_MINOR_VERSION, 6) {
```

Comment:
...so how would a Qt4 `qmake` respond to this - if it errors with say "unknown command lessThen() at line:21 in ./src/src.pro" then that will (___not___?) work but not quite in the manner anticipated! :stuck_out_tongue_winking_eye: